### DISCUSSION TITLE
add social preview in godot repository

### DISCUSSION THREAD

**Discussion by bossoverdose** [godotengine/godot-proposals#6256] [General Discussions]
godot don't have a image, like bevy https://github.com/topics/game-engine

add image in repository > settings > general > social preview

![1280x640](https://user-images.githubusercontent.com/28818773/217821320-fa95159e-d223-4c56-9725-258992bc72bb.png)
(👆 this image is in the right size 1280x640)

**Comment by coppolaemilio** [comment]
Seems like a good idea! I think we should be able to make one for the repository 👍
